packages/client/src/components/AccountViews/PasswordView.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import {
+  View,
+  Text,
+  TextInput,
+  ScrollView,
+  StyleSheet,
+  Image,
+  Dimensions,
+  Button,
+} from 'react-native'
+
+const width = Dimensions.get('window').width
+const height = Dimensions.get('window').height
+
+const AccountView = () => {
+  const [password, setPassword] = useState('')
+  const [confirmPassword, setConfirmPassword] = useState('')
+
+  return (
+    <ScrollView>
+      <LabelInput
+        label='Password'
+        placeholder='Please enter your password'
+        value={(text) => setPassword(text)}
+        secureText
+      />
+      <LabelInput
+        label='Confirm Password'
+        placeholder='Please re-enter your password'
+        value={(text) => setConfirmPassword(text)}
+        secureText
+      />
+      <Button
+        onPress={confirmPasswordsMatch(password, confirmPassword)}
+        title='Change Password'
+        color='#841584'
+        accessibilityLabel='Learn more about this purple button'
+      />
+      <Image
+        style={styles.bottomImageStyle}
+        source={require('../../../assets/Bottom_Image_Styling.png')}
+      />
+    </ScrollView>
+  )
+}
+
+const LabelInput = (props) => (
```

Comment:
Separate file for separate view?

Reply:
Why is this defined here **and** in a separate file???